docs/admin/config.yml

```diff
@@ -469,10 +472,11 @@ collections:
     create: true
     delete: true
     slug: '{{slug}}'
+    identifier_field: title
```

Comment:
Do we need this? Isn't `title` the default and you'd set `identifier_field` if you were using a different field?

Reply:
If the title field isn't a `string`, the cms throws an error. This is because they're worried about people using something like a `markdown` field which would be difficult to convert into an identifier string. The `identifier_field` assuages the cms' anxiety and affirms that our `string-trimmed` widget is safe.